WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/User/UserPropertiesSetEventDecoder.swift

```diff
@@ -74,7 +74,7 @@ struct UserPropertiesSetEventDecoder {
 
 }
 
-struct LabelsPayload: Decodable {
+struct LabelsPayloadV0: Decodable {
```

Comment:
Update events are not versioned, so I think making the payload versioned might cause confusion. To maintain separation between versioned APIs and update events, I would leave this payload is it was, and duplicated it as `LabelsPayloadV0` within the user properties api.

Reply:
Ok good to know thanks, I'll have to do some changes in my other PR also